blank-typescript/template/src/App.tsx

```diff
@@ -54,8 +58,8 @@ export const App = (): JSX.Element => {
     return (
         <div className={classes.pageBackground}>
             <AppBar position={'fixed'}>
-                <Toolbar>
-                    <IconButton color={'inherit'} edge={'start'}>
+                <Toolbar className={classes.toolbar}>
+                    <IconButton color={'inherit'} edge={'start'} style={{ marginRight: theme.spacing(4) }}>
```

Comment:
```suggestion
                    <IconButton color={'inherit'} edge={'start'} style={{ marginRight: theme.spacing(3) }}>
```

![image](https://user-images.githubusercontent.com/8997218/99567385-17734780-299c-11eb-9b16-ea42743e667e.png)

Reply:
Same for the js side

Reply:
mm, i thought it looked big...didn't account for the 8px padding on the icon.